optuna_dashboard/preferential/samplers/__init__.py

```diff
@@ -0,0 +1,6 @@
+from __future__ import annotations
+
+from ._gp import PreferentialGPSampler
+
+
+__all__ = ["PreferentialGPSampler"]
```

Comment:
Unlike Optuna, Optuna Dashboard does not have a lazy importing function. So I think it's better to avoid importing PreferentialGPSampler here since it requires botorch, which is an optional dependency.

Does it sound reasonable to import PreferentialGPSampler via `from optuna_dashboard.preferential.gp import PreferentialGPSampler`?